files-to-sync/.github/workflows/pr-notification.yml

```diff
@@ -0,0 +1,72 @@
+# This file is managed by macpro-mdct-core so if you'd like to change it let's do it there
```

Comment:
This is so great to get in this sync workflow! It was on my someday figure out how to do this list.